tests/fixtures/saas/zendesk_fixtures.py

```diff
@@ -88,3 +100,66 @@ def zendesk_dataset_config(
     )
     yield dataset
     dataset.delete(db=db)
+
+
+@pytest.fixture(scope="function")
+def zendesk_create_erasure_data(
+    zendesk_connection_config: ConnectionConfig, zendesk_erasure_identity_email: str
+) -> None:
+
+    zendesk_secrets = zendesk_connection_config.secrets
+    base_url = f"https://{zendesk_secrets['domain']}"
+    headers = {
+        "Content-Type": "application/json",
+        "Authorization": "Basic {}".format(
+            base64.b64encode(
+                bytes(
+                    f"{zendesk_secrets['username']}:{zendesk_secrets['api_key']}",
+                    "utf-8",
+                )
+            ).decode("ascii")
+        ),
+    }
+
+    connector = SaaSConnector(zendesk_connection_config)
+
+    # user
+    body = json.dumps(
+        {
+            "user": {
+                "name": "Ethyca Test Erasure",
+                "email": zendesk_erasure_identity_email,
+                "verified": "true",
+            }
+        }
+    )
+    updated_headers, formatted_body = format_body({}, body)
```

Comment:
We don't need to call `json.dumps` or `format_body` we can just use the user dict directly like you did on line 150-161.